cmd/trace-agent/subcommands/run/command_test.go

```diff
@@ -0,0 +1,18 @@
+package run
+
+import (
+	"context"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/cmd/trace-agent/subcommands"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+func TestFxRun(t *testing.T) {
+	fxutil.TestRun(t, func() error {
+		ctx := context.Background()
+		cliParams := RunParams{GlobalParams: &subcommands.GlobalParams{}}
+		defaultConfPath := ""
+		return runFx(ctx, &cliParams, defaultConfPath)
```

Comment:
nit: could we rename `runFx` with just run `run`

I know the name of the function was like that before, but just suggesting removing information about the underlying system from the function names

Reply:
This was a bit trickier, because there's some cross-platform build rules around the caller, but I think the new renames are better.